src/axom/CMakeLists.txt

```diff
@@ -48,7 +48,6 @@ install(TARGETS              axom
         DESTINATION          lib)
```

Comment:
I don't think these BLT changes would allow for the empty source file to be removed, but I am working on some internal object library changes in BLT that may eliminate the need for this.